crates/common_utils/src/lib.rs

```diff
@@ -20,13 +20,25 @@ pub mod date_time {
     #[cfg(feature = "async_ext")]
     use time::Instant;
     use time::{
-        format_description::well_known::iso8601::{Config, EncodedConfig, Iso8601, TimePrecision},
+        format_description::{
+            well_known::iso8601::{Config, EncodedConfig, Iso8601, TimePrecision},
+            FormatItem,
+        },
         OffsetDateTime, PrimitiveDateTime,
     };
     /// Struct to represent milliseconds in time sensitive data fields
     #[derive(Debug)]
     pub struct Milliseconds(i32);
 
+    /// Enum to represent date formats
+    #[derive(Debug)]
+    pub enum DateFormat {
+        /// Format the date in 20191105081132 format
+        YYYYMMDDHHmmss,
+        /// Format the date in 20191105 format
+        YYYYMMDD,
+    }
```

Comment:
Do we need this as a enum? can't we make a type out of it?. Probably make use of strategy pattern, and have a transparent type, that will store the date in the specified format.